### ISSUE TITLE
`Publish` component default `publishedAt` has bad default format 

### ISSUE THREAD

**Issue by senecolas** [BRACKETS-by-TRIAD/craftable-pro-docs#23]
## Issue
In listing, when I click on "Publish" and I don't edit the datepicker timestamp, a date in ISO8601 format is send to the update url and causes an sql error:

```
SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: '2023-04-07T11:06:27.744Z' for column 'published_at' at row 1 (Connection: mysql, SQL: update `references` set `published_at` = 2023-04-07T11:06:27.744Z, `references`.`updated_at` = 2023-04-07 11:08:27 where `id` = 3)
```

**Comment by timoransky** [comment]
Hi @senecolas 
Thank you for the bug report, I will take a look into this one as well.

Will keep you updated 😉

**Comment by timoransky** [comment]
Hi @senecolas 
We just released a new version ([1.1.7](https://docs.craftable.pro/changelog#v117---2023-04-13)) where this bug is also fixed 🤗
